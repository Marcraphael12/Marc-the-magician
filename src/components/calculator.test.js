/* eslint-disable no-unused-vars */
import React from 'react';
import renderer from 'react-test-renderer';
import {
  render,
} from '@testing-library/react';
import MakeOperations from './calculator';

it('calculator snapshot test', () => {
  const tree = renderer
    .create(<MakeOperations />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

render(<MakeOperations />);
