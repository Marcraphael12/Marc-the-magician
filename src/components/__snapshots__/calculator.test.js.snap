// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculator snapshot test 1`] = `
<div
  className="calc"
>
  <h2>
    Let's do some math
  </h2>
  <ul>
    <li
      className="screen"
    >
      <h3>
        
      </h3>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        %
      </button>
    </li>
    <li
      className="pad different"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        7
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        8
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        9
      </button>
    </li>
    <li
      className="pad different"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        4
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        5
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        6
      </button>
    </li>
    <li
      className="pad different"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        1
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        2
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        3
      </button>
    </li>
    <li
      className="pad different"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </li>
    <li
      className="pad zero"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        0
      </button>
    </li>
    <li
      className="pad "
    >
      <button
        onClick={[Function]}
        type="button"
      >
        .
      </button>
    </li>
    <li
      className="pad different"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </li>
  </ul>
</div>
`;
